repos/system_upgrade/el7toel8/actors/selinux/selinuxcontentscanner/tests/component_test.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+import pytest
+
+from leapp.snactor.fixture import current_actor_context
+from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
+from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.reporting import Report
+
+test_modules = [
+    ["400", "mock1"],
+    ["99", "mock1"],
+    ["200", "mock1"],
+    ["400", "mock2"],
+    ["999", "mock3"]
+]
+
+semanage_commands = [
+    ['fcontext', '-t', 'httpd_sys_content_t', '"/web(/.*)?"'],
+    ['fcontext', '-t', 'ganesha_var_run_t', '"/ganesha(/.*)?"'],
+    ['port', '-t', 'http_port_t', '-p', 'udp', '81'],
+    ['permissive', 'abrt_t']
+]
+
+testmoduledir = os.path.join(os.getcwd(), "tests/mock_modules/")
+
+
+def setup_():
```

Comment:
I'm really surprised pytest allows setup_/teardown_ for names. Please follow their naming convention and add a module name after _, like https://docs.pytest.org/en/latest/xunit_setup.html